script/options.js

```diff
@@ -115,6 +115,10 @@ function applyBrandConsolidationOverrides(options) {
     collections: require('./collections/brand-consolidation.json'),
     redirects: require('./vagovRedirects.json'),
     domainReplacements,
+    contentAssets: {
+      source: path.join(options['content-directory'], '../assets'),
+      destination: './',
+    },
```

Comment:
I don't think it hurts anything, but should this behind the WBC feature flag? `options['content-directory']` right now is only a WBC argument.

Reply:
Yep, this is inside the `applyBrandConsolidationOverrides`